app.js

```diff
@@ -1,11 +1,18 @@
-import { fetchGithubRepos } from './GithubApi.js';
+import { fetchGithubIssues, fetchGithubRepos } from './GithubApi.js';
+import { formatDateTime } from './utils.js';
 
 let form = document.querySelector('form');
 let dialog = document.querySelector('.details-dialog');
-let dialogContent = dialog.querySelector('.details-dialog__content');
+let dialogContent = dialog.querySelector('.details-dialog__content__dynamic');
 let dinamicContent = document.querySelector('#dinamicContent');
 let skeletonScreen = document.querySelector('.skeleton-screen');
 
+dialog.addEventListener('click', (event) => {
+    if (event.target === dialog) {
```

Comment:
Here it's worth adding a comment that explains why this works. If you come back later to this code you might be a bit confused.